src/app/components/crypto-asset-item/crypto-asset-item.layout.tsx

```diff
@@ -79,16 +89,15 @@ export function CryptoAssetItemLayout({
     />
   );
 
-  if (isInteractive)
-    return (
-      <Pressable
-        data-testid={dataTestId}
-        onClick={() => onSelectAsset(availableBalance.symbol, contractId)}
-        my="space.02"
-      >
-        {content}
-      </Pressable>
-    );
-
-  return <Box my="space.02">{content}</Box>;
+  return isInteractive ? (
+    <Pressable
+      data-testid={dataTestId}
+      onClick={() => onSelectAsset(symbol, contractId)}
+      my="space.02"
+    >
+      {content}
+    </Pressable>
+  ) : (
+    <Box my="space.02">{content}</Box>
+  );
```

Comment:
Early return is a more readable pattern than this ternary, suggest reverting this change